Dockerfile

```diff
@@ -1,11 +1,14 @@
 FROM node:18.12.1-alpine as builder
 
+ARG SENTRY_AUTH_TOKEN
+ARG SENTRY_DSN=https://3357d5dab93b4a88bea7f0ea4a6439f2@o1200558.ingest.sentry.io/4504750561034240
```

Comment:
It's better to be configurable

Reply:
Here is a default value similar to the following `API_URL`, `CHAIN_TYPE`, but I did forget to specify that you also need to provide `SENTRY_DSN`.